test/functional/fixtures/regression/gh-1054/test.js

```diff
@@ -0,0 +1,11 @@
+describe('[Regression](GH-1054)', function () {
+    describe('Target element should have the first symbol of typing text when the first input event raised', function () {
+        it("Typing in the input element with 'replace' option", function () {
+            return runTests('testcafe-fixtures/index.test.js', 'Type text in the textarea');
```

Comment:
` it("Typing in the input element with 'replace' option", ...`
`...  'Type text in the textarea');`

??